src/views/dropdown.rs

```diff
@@ -30,28 +31,120 @@ prop_extractor!(DropdownStyle {
     close_on_accept: CloseOnAccept,
 });
 
-pub fn dropdown<T, MF, I, LF, AIF>(
-    active_item: AIF,
-    main_view: MF,
-    iterator: I,
-    list_item_fn: LF,
-) -> Dropdown<T>
-where
-    MF: Fn(T) -> Box<dyn View> + 'static,
-    I: IntoIterator<Item = T> + Clone + 'static,
-    LF: Fn(T) -> Box<dyn View> + Clone + 'static,
-    T: Clone + 'static,
-    AIF: Fn() -> T + 'static,
-{
-    Dropdown::new(active_item, main_view, iterator, list_item_fn)
-}
-
-/// A dropdown widget
+/// Create a dropdown widget.
+///
+/// # Example
+///
+/// A common scenario is populating a dropdown menu using an enum.
```

Comment:
I think it would be good to have this in the docs for the Dropdown struct. The example in the widget gallery for the dropdown does exactly this so it might be good to have the docs also reference that example.